src/components/schemas/Property-Marketing.yaml

```
@@ -167,3 +167,6 @@ properties:
       - TAX_FREE
   owner:
     $ref: ./Owner.yaml
+  marketStress:
+    $ref: ./MarketStress.yaml
+    description: Renseigné uniquement sur les biens de source=`YANPORT`
```

:hammer_and_wrench: issue: Le champ `description` n'a aucun effet ici puisque tu utilises une référence au dessus. Il faut déplacer ce champ dans le fichier `MarketStress.yaml`. Aussi je suggère d'enlever le signe = :
```suggestion
    description: Renseigné uniquement sur les biens de source `YANPORT`
```